11/collection-meta.yaml

```diff
@@ -235,8 +235,9 @@ collections:
       - Shilpi-J
     repository: https://github.com/ansible-collections/ibm.spectrum_virtualize
     removal:
-      major_version: TBD
+      major_version: 12
       reason: renamed
+      announce_version: 11.0.0
```

Comment:
This means 11.0.0 wouldn't be _exactly_ the same as 11.0.0rc1. I'm not sure if this is really a problem. On the other hand: Why not wait until 11.1.0? It's only 2 weeks later. And since we want to release 11.0.0 in two days, this is on very short notice

Reply:
Well, 11.0.0 is NEVER exacly like 11.0.0rc1, since 1) it has a different version, and 2) the changelog and porting guide look very different. For example compare the changelog of 10.0.0rc1 (https://github.com/ansible-community/ansible-build-data/blob/10.0.0rc1/10/CHANGELOG-v10.md#v10-0-0) with the one of 10.0.0 (https://github.com/ansible-community/ansible-build-data/blob/10.0.0/10/CHANGELOG-v10.md). So having an additional changelog entry announcing the removal isn't a big difference as opposed to all the other changes.

Reply:
It depends, either this gets enough approval and we can merge it before the 11.0.0 release, or I'll bump the version here and we merge it for 11.1.0. Or later, if there aren't enough approvals until then either.